micro_ros_agent/src/agent/graph_manager/graph_manager.cpp

```diff
@@ -355,7 +358,10 @@ void GraphManager::add_datawriter(
 {
     const std::string& topic_name = datawriter->get_topic()->get_name();
     const std::string& type_name = datawriter->get_topic()->get_type_name();
-    this->add_datawriter(datawriter_guid, topic_name, type_name,
+
+    auto remapped_topic_name = namespace_remapping_ + "/" + topic_name;
+
+    this->add_datawriter(datawriter_guid, remapped_topic_name, type_name,
```

Comment:
I think this is a mistake here?  the remapping should only be in the overload of `add_datawriter` below?  This will double namespace it?

Reply:
probably it was intended for `add_datareader`?  Which neither overload has a remapping